girder/api/rest.py

```diff
@@ -1057,10 +1057,22 @@ class boundHandler(object):  # noqa: class name
     ``self.boolParam``, ``self.requireParams``, etc.
     """
     def __init__(self, ctx=None):
-        self.ctx = ctx or _sharedContext
+        if callable(ctx):  # Used as a raw decorator
```

Comment:
@salamb How about flipping the logic, and check if the passed is an instance of Resource or a subclass?